datadog/src/main/scala/zio/metrics/connectors/datadog/DatadogEncoder.scala

```diff
@@ -41,5 +41,8 @@ case object DatadogEncoder {
     (key, values) => Chunk.fromArray(withContainerId(key, values).toString().getBytes())
   }
 
+  private def makeStatsdEncoder(config: DatadogConfig): StatsdEncoder =
+    StatsdEncoder(config.entityId.map(eid => s"dd.internal.entity_id:$eid"))
```

Comment:
I would do the string formatting in `StatsdEncoder`s constructor so all the formatting logic is in one place

Reply:
Makes sense!

Reply:
https://github.com/zio/zio-metrics-connectors/pull/58/commits/5aeca23427d1cbcf40829fcf8db6d82a48aa755a